setup.py

```diff
@@ -84,7 +84,8 @@ def readme():
         # Until this issue is closed
         # https://github.com/googleapis/google-cloud-python/issues/10566
         "google-auth>=1.25.0,<3.0.0dev",  # Work around pip wack.
-        "google-cloud-bigquery>=2.25.2,<3.0.0dev",
+        "google-cloud-bigquery>=2.25.2,<4.0.0dev",
+        "google-cloud-bigquery-storage>=2.0.0,<3.0.0dev",
```

Comment:
Need pyarrow as well. Probably a good thing we're adding this, because I totally missed that.